packages/tasit-action/src/NFT.js

```diff
@@ -0,0 +1,44 @@
+import Contract from "./Contract";
+
+class ABIs {
+  static getERC721Full = () => {
+    // TODO: Is there a better way to do that for now?
+    return require("./abi/ERC721Full.json");
+  };
+}
+
+export class NFT {
+  #contract;
+  #abi;
+
+  constructor(address, wallet) {
+    this.#abi = ABIs.getERC721Full();
+    this.#contract = new Contract(address, this.#abi, wallet);
+    this.#addFunctionsToNFT();
+  }
+
+  getAddress = () => {
```

Comment:
Wow, thanks! You are right. The NFT.js file has now <20 lines.

Reply:
Hmm yes, interesting. Since we're dynamically adding the functions to the contract using the ABI, it seems that the NFT abstraction may do less additional work than I thought. I'll keep thinking about this.

Reply:
We can include js higher-level convenience methods as a result of what the ABI contains.

`isMintable`, `isPausable`, `isBurnable`, `isEnumerable`, `hasMetadata`, `isFull` (enumerable and has metadata)

https://github.com/OpenZeppelin/openzeppelin-solidity/tree/master/contracts/token/ERC721

Reply:
Yeah: https://github.com/tasitlabs/TasitSDK/issues/146

Reply:
Thanks. Resolving